ctapipe/tools/fileinfo.py

```diff
@@ -0,0 +1,100 @@
+"""
+Display information about ctapipe output files (DL1 or DL2)
+"""
+
+from pathlib import Path
+import tables
+import yaml
+from astropy.table import Table
+from ctapipe.tools.utils import get_parser
+
+
+def unflatten(dictionary, separator=" "):
```

Comment:
Is this useful outside this tool? Maybe directly add this to `ctapipe.io.metadata`?